tests/acceptance/features/apiFullTextSearch/search.feature

```diff
@@ -83,3 +84,20 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search restored version of a file by content
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "test.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "test.txt"
+    And user "Alice" has restored version index "1" of file "test.txt"
+    When user "Alice" searches for "Content:hello" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
+      | test.txt |
+    And the content of file "test.txt" for user "Alice" should be "hello world"
```

Comment:
this scenario is about search so we don't need this step in this scenario i think we already have different scenario which cover checking restored fileversion content
```suggestion
    And the content of file "test.txt" for user "Alice" should be "hello world"
```

Reply:
I have got suggestion to add this step here https://github.com/owncloud/ocis/pull/6970#discussion_r1286568314. Should I keep it or remove it? @SagarGi

Reply:
that ok. but we don't need this step here because this scenario is for search not for restore